.gitignore

```diff
@@ -1,13 +1,20 @@
 sandbox/
 *.bak
+grafana/alerting/
+grafana/csv/
 grafana/grafana.db
 grafana/plugins/
 influxdb/meta/
 influxdb/wal/
 influxdb/data/
+compose.env
+grafana.env
 pypowerwall.env
+telegraf.local
```

Comment:
Looks correct to me, however will wait for Jason to confirm, as well as for the other suggestions below.

I guess the question could be what/how many files to ignore vs. being happy with seeing them as untracked?

Reply:
sure.. especially if the contents of those files need to be manually adjusted periodically. in that case, NOT adding these three files and maybe removing pypowerwall.env is better.

My guess is that it's better to add them, but I'm fairly new to the project.

Reply:
I'm leaning towards adding them to .gitignore as the best option, _including_ the `influxdb/run-once-*.sql.done` files.

My reasoning/thoughts on this (I'm open to counter opinions or pros/cons of course):

The .env files that are considered part of the core project and a requirement for the system to work, generally should be included in .gitignore

Also adding the `influxdb/run-once-*.sql.done` files to .gitignore may be advisable too, since if these are seen as an "untracked" file by a user, potentially they may think they are redundant files and delete them... which means they will get re-run again whenever an upgrade is initiated.

Reply:
I agree!  Good call on this @mcbirse , including the *done files!